.checkpatch.conf

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright 2017, Linaro Ltd and contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Do not expect a Linux tree
+--no-tree
+
+# Allow lines up to 90 characters long
+--max-line-length=90
```

Comment:
Usually it is 80 characters that people recommend to use (old terminal default size).

Reply:
Yeah, I thought maybe we could be a bit liberal about it, as this comes from the very old days. Yet, if you think there's good reason to stick to 80 chars, I'm not too fussed about it :-)

Reply:
I am, that's why I merged it even after my comment :)